pkg/approver/approver.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package approver
+
+import (
+	"context"
+
+	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+
+	cmpapi "github.com/cert-manager/policy-approver/pkg/apis/policy/v1alpha1"
+)
+
+// Interface is an Approver. Implements Evaluator.
```

Comment:
> Interface is an Approver
Note: this may be a little confusing because there is no `Approver` type and 'evaluator' and 'approver' seem similar terms. To me it seems like it is the manager that would fit the 'approver' concept since it calls all the evaluators?

Reply:
An `Approver` is meant to encompass not just an `Evaluator`, but also controller implementations, validation, readiness etc. An Approver manager wraps multiple Approvers.

Perhaps the `Approver Manager` should be renamed to `Evaluator Manager`?

Reply:
> Approvers` will have other member funcs/interfaces
I think that it will make more sense at that point then.
I am guessing that if the `Approver manager` wraps multiple approvers which are more than just evaluators then the name is ok 👍🏼